### PR TITLE
display and update real user data on account screens

### PR DESCRIPTION
**Pull request by jwietelmann**
connect #243 

* Removes password input. (Passing a new password to `update` does not appear to change the password in the API at this time.)
* Displays real user attributes from the API.
* Saves changes to user attributes to the API.
* Alerts on success.
* Alerts on validation failures.
* Updates `currentUser` in auth state on success.

### CODE REVIEW

File: src/Accounts/account.js [hunk 1]
```diff
@@ -1,104 +1,168 @@
...
+
+          <View style={accountStyles.inputContainer}>
+            <View style={accountStyles.row}>
+              <Text style={accountStyles.accountInputHeaderDisabled}>First Name</Text>
+              <TextInput
+                style={accountStyles.accountInputHeader}
+                defaultValue={user.first_name}
+                placeholderTextColor='#CCC'
+                underlineColorAndroid="transparent"
+                onChangeText={this.updateUser('first_name')}
```
Comment (jwietelmann) [inline src/Accounts/account.js]: Most of the changes in the render function are merely indentation changes that resulted from me turning it from a function component to a Component class.

The `onChangeText` props are the only real change in the JSX.

**Comment by jwietelmann** [comment]
Good call on the doorman thing. Let me go see if I can knock that out.

**Comment by zmelamed** [comment]
There's actually a special endpoint for retrieving the events that can be displayed to a user who has doorman permissions, this is being built this week (a user wont be able to see every event if they have doorman persmissions, they'll only be able to see the specific events for the venue that they work for): https://github.com/big-neon/bn-api/issues/591

**Comment by jwietelmann** [comment]
@daybreaker Conflicts are fixed. I need to figure out how to become a doorman before I can tell you if the setting is coming back from the server or not. Let's just ship this, and I won't move the ticket to QA yet.

**Pull request merged**